src/pudl/metadata/resources/eia860.py

```diff
@@ -642,6 +642,68 @@
         "sources": ["eia860"],
         "etl_group": "eia860",
     },
+    "_core_eia860__cooling_equipment": {
+        "description": (
+            "Information about cooling equipment at generation facilities, "
+            "from EIA-860 Schedule 6D. Currently in an intermediate state, "
+            "use at your own risk."
+        ),
+        "schema": {
+            "fields": [
+                "plant_id_eia",
```

Comment:
Nice to have: organize columns roughly by content - e.g., put county and state next to one another, utility ID and name together, etc. `report_date` should be first to be consistent with other data sets.